arangograph/backups.md

```diff
@@ -38,20 +46,58 @@ It's also possible to create a backup on demand. To do this, click **Back up now
 
 ![Back up Now Dialog](images/arangograph-back-up-now-dialog.png)
 
+If you want to manually copy a backup to a different region than the default
+one, first ensure that the **Upload backup to storage** option is enabled.
+Then, highlight the backup row and use the
+**Copy backup to a different region** button from the **Actions** column. 
```

Comment:
enterprise only

Reply:
Then we need to say it. Not necessarily make a note, but at least mention in parentheses that it's enterprise only. Otherwise we're forcing users to create a connection between seemingly unconnected text sections. Users can only read one of the sections and not the entire page, but they still should be able to get the necessary info.